packages/react-examples/src/react-charting/GaugeChart/GaugeChart.Basic.Example.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { GaugeChart } from '@fluentui/react-charting';
+
+export class GaugeChartBasicExample extends React.Component {
+  public render(): React.ReactNode {
+    return (
+      <GaugeChart
+        segments={[
+          { size: 33, color: '#C50F1F' },
```

Comment:
This example doesn't show up in the demo site

Reply:
fixed